runners/console/index.ts

```diff
@@ -80,6 +82,9 @@ export class Console extends Runner {
             this.executeCommandSync("bash setup " + path.join(settingsDir, "settings.git").replace(/\\/g, "/"), installDir, result, "yes");
         }
 
+        this.setVariable(this.workspaceDirectory, path.join(this.getWorkingDirectory(), "devonfw", "workspaces", "main"));
```

Comment:
I think you should use the new way to get the workspace directory in runCreateDevon4jProject, assertCreateDevon4jProject and assertCobiGenJava, too. 
And maybe is it useful to log a warning when a command is used which requires the devon ide, but it's not used.

Reply:
Wouldn't it be better to just add the warning without changing the paths. My thoughts were, that the given path is mandatory for the function to work. So it will throw an Error, when the directories don't exist.

Reply:
Then you should revert it in runCobiGenJava. There you handle it the new way.
My thoughts were that if no ide is installed the noErrorCode assertion will throw the error because the devon command is not recognized. So the directories will not be checked at all.